mantidproject/mantid
# Use new correction algorithms in IDA Calc/ApplyCorr

Fixes [#11326](http://trac.mantidproject.org/mantid/ticket/11326).

To test:
- Review the CalcCorr and ApplyCorr documentation (`?` button)
- Try running with some combinations of data and corrections below (will need to be on Windows in Release build for Cylinder)

Data: Reduced (meV)
- Sample: `irs26176_graphite002_red.nxs`
- Can: `irs26173_graphite002_red.nxs`

Data: Reduced (A)
- Sample: `wl_irs26176_graphite002_red.nxs`
- Can: `wl_irs26173_graphite002_red.nxs`

Data: S(Q, w)
- Sample: `irs26176_graphite002_sqw.nxs`
- Can: `irs26173_graphite002_sqw.nxs`

Data: Diffraction
- Sample: `irs26176_diffspec_red.nxs`
- Can: `irs26173_diffspec_red.nxs`

Corrections: Flat Plate
- Sample Thickness: 0.01
- Sample Angle: 45
- Can Front: 0.1
- Can Back: 0.1

Corrections: Cylinder
- Can Inner: 0.19
- Sample Inner: 0.2
- Sample Outer: 0.25
- Can Outer: 0.26
- Beam Height/Width: 2.1 (should be populated automatically for most data)
- Step Size: 0.002

Common Options
- Sample Number Density: 0.1
- Sample Chemical Formula: H2-O
- Can Number Density: 0.1
- Can Chemical Formula: V

There is a Mantid project in `babylon5/Public/DanNixon/abscor` that will load all the sample data.
